contracts/lib/zeppelin/token/BasicToken.sol

```diff
@@ -11,9 +11,11 @@ import '../math/SafeMath.sol';
  */
 contract BasicToken is ERC20Basic {
   using SafeMath for uint256;
+  uint256 totalSupply_;
 
   mapping(address => uint256) balances;
 
+  function totalSupply() public view returns (uint256) { return totalSupply_; }
```

Comment:
These `totalSupply` changes are copy pasta'd from zeppelin, yeah? (I forget where they had this implemented before, but I guess not here?)

Reply:
copy pasta indeed